upstairs/src/lib.rs

```diff
@@ -4305,13 +4298,15 @@ impl Upstairs {
          * Build the flush request, and take note of the request ID that
          * will be assigned to this new piece of work.
          */
+        let ddef = self.ddef.lock().await;
         let fl = create_flush(
             next_id,
             dep,
             next_flush,
             gw_id,
             self.get_generation().await,
             snapshot_details,
+            ImpactedBlocks::new(*ddef),
```

Comment:
There are no "impacted blocks" for flushes, right?  Maybe a future improvement is to not need to get that ddef.lock() and just send some "default empty" ImpactedBlocks?

Reply:
right, flushes don't "impact" any blocks on the disk